From 03744e21ae0f7a041c96bc4fb5fc8b1cdc43efe9 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Wed, 14 Aug 2019 11:12:45 -0400
Subject: [PATCH] MANTA-4487 manta-compute-bin transitive dtrace-provider deps
 too old for x86_64 MANTA-4493 manta-compute-bin tests can not run on any
 version of node

---
 package.json | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/package.json b/package.json
index f158a5c..66c306a 100644
--- a/package.json
+++ b/package.json
@@ -1,15 +1,15 @@
 {
     "name": "manta-compute-bin",
     "description": "Utilities included in Manta compute.",
-    "version": "1.0.0",
+    "version": "1.1.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "main": "lib/index.js",
     "dependencies": {
         "assert-plus": "0.1.2",
-        "bunyan": "0.20.0",
+        "bunyan": "1.8.12",
         "lstream": "0.0.4",
-        "manta": "^3.0.0",
+        "manta": "^5.0.0",
         "memorystream": "0.2.0",
         "node-uuid": "1.4.0",
         "posix-getopt": "1.0.0",
@@ -19,7 +19,7 @@
         "verror": "1.3.6"
     },
     "devDependencies": {
-        "nodeunit": "0.8.0"
+        "nodeunit": "0.9.1"
     },
     "directories": {
         "bin": "./bin",
-- 
2.21.0

